src/photos/photo.controller.ts

```diff
@@ -0,0 +1,17 @@
+import { Controller, Get } from '@nestjs/common';
+import { PhotoService } from './photo.service';
+
+@Controller('photos')
+export class PhotoController {
+  constructor(private readonly photoService: PhotoService) {}
+
+  @Get()
+  async getPhotos(): Promise<any> {
```

Comment:
Probably return type of Promise will be something like Promise<Array<Photo>>
In the future better to try not to use **any**, only in extraordinary cases.